config/database.yml

```diff
@@ -1,18 +1,21 @@
-default: &default
```

Comment:
why are we exposing our database_name, password, host, port?

read it from credential file or we can use a simple gem figaro to set ENV variables and we can read env variables here